src/librbd/image/OpenRequest.cc

```diff
@@ -359,6 +359,85 @@ Context *OpenRequest<I>::handle_v2_get_create_timestamp(int *result) {
     return nullptr;
   }
 
+  send_v2_get_access_timestamp();
+  return nullptr;
+}
+
+template <typename I>
+void OpenRequest<I>::send_v2_get_access_timestamp() {
+  CephContext *cct = m_image_ctx->cct;
+  ldout(cct, 10) << this << " " << __func__ << dendl;
+
+  librados::ObjectReadOperation op;
+  cls_client::get_access_timestamp_start(&op);
```

Comment:
Nit: since access and modified timestamps were added at the same time, you can combine the two ops in a single call to avoid an extra round-trip to the server.

Reply:
@dillaman I pushed 2 commits (2d6ac7c & a9dd9fb) addressing this issue, is this what you expected? Do you think including also the create timestamp in the call to save another round-trip to the server would be interesting?

Reply:
@colletj You don't need to merge the two ```cls_client``` methods together -- you can just do something like:

```
librados::ObjectReadOperation op;
cls_client::get_access_timestamp_start(&op);
cls_client::get_modify_timestamp_start(&op);
```

Reply:
... as for your other question, we won't be able to merge the ```cls_client::get_create_timestamp_start``` call until the Luminous release is EoLed. It might be nice to leave a short ```TODO: merge w/ create timestamp query after luminous EOLed``` comment.